example/android/app/src/main/kotlin/com/ryanheise/example/MainActivity.kt

```diff
@@ -0,0 +1,6 @@
+package com.ryanheise.example
+
+import io.flutter.embedding.android.FlutterActivity
+
+class MainActivity: FlutterActivity() {
+}
```

Comment:
Probably shouldn't be adding Kotlin code for a web-centric PR. Also I would note that:

1. This example was already configured to use Java rather than Kotlin, and
2. Ideally we can eventually remove even the Java MainActivity class altogether (not as part of this PR, though).

Reply:
Yeah should have removed that from the commit... I ran flutter create to have a web version of example, I guess it had side effects haha

Reply:
@ryanheise Not sure how the example app would work without a main activity. Is that possible?

Reply:
Yes it's possible (although not as part of this PR). See:

https://github.com/flutter/flutter/wiki/Upgrading-pre-1.12-Android-projects

>  the body of the MainActivity is now empty, you can also optionally delete the MainActivity.java/kt file if you'd like.